core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-results-seo-tool/dot-results-seo-tool.component.spec.ts

```diff
@@ -102,18 +100,38 @@ describe('DotResultsSeoToolComponent', () => {
     });
 
     it('should display mobile size for the preview', () => {
-        const previews = spectator.queryAll(byTestId('seo-preview'));
+        spectator.setInput({
+            seoMedia: SEO_MEDIA_TYPES.GOOGLE
+        });
+        spectator.detectChanges();
+        const previews = spectator.queryAll(byTestId('seo-preview-mobile'));
         expect(previews[1]).toHaveClass('results-seo-tool__version--small');
     });
 
-    it('should filter seo results by seoMedia on changes', () => {
-        spectator.fixture.componentInstance.seoMedia = 'facebook';
-        spectator.component.ngOnChanges();
-        spectator.component.currentResults.subscribe((items) => {
+    it('should filter seo results by Facebook, seoMedia on changes', () => {
+        spectator.setInput({
+            seoMedia: SEO_MEDIA_TYPES.FACEBOOK
+        });
+        spectator.detectChanges();
+        spectator.component.currentResults$.subscribe((items) => {
```

Comment:
remember to use done() when you subscribe in a test. If not can have false positives.